lib/shortcuts/event-lambda.js

```diff
@@ -41,6 +73,7 @@ class EventLambda extends Lambda {
 
     const {
       EventPattern,
+      EventBusName = 'default',
```

Comment:
AWS also defaults to 'default' and this can also be an ARN, but I thought we should be explicit here. We can not default at all here and just only set this if user passes it in.